zproject/prod_settings_template.py

```diff
@@ -72,10 +72,12 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 # The email From address to be used for automatically generated emails
-DEFAULT_FROM_EMAIL = "Zulip <zulip@example.com>"
+# (e.g. "Zulip <zulip@example.com>")
+DEFAULT_FROM_EMAIL = "Zulip <zulip@{0}>".format(EXTERNAL_HOST.split(":")[0])
```

Comment:
Do you think that `DEFAULT_FROM_EMAIL` should perhaps use ZULIP_ADMINISTRATOR as the email address instead?  The point of this is that it should be an address that actually receives mail, and that's what is guaranteed to.